### PR TITLE
8.x 1.0 beta1 add video

### PR DESCRIPTION
**Pull request by mitchellkeaney**
I've added a link to documentation contributed by Zach Spalding of how to install Archipelago on an AWS Ubuntu server.

### CODE REVIEW

File: docs/ubuntu.md [hunk 1]
```diff
@@ -174,6 +176,9 @@ If you see any issues or errors or need help with a step, please let us know (AS
 
 This is our first beta release and even when we have a longer Path to travel until we feel we are ready for a full `8.x-1.0`, we are pretty excited about how far we have gotten in less than a year. If you like this, let us know.
 
+### User contributed documentation:
+_Insatlling Archipelago on AWS Ubuntu_ by Zach Spalding: https://youtu.be/RBy7UMxSmyQ
```
Comment (DiegoPino) [inline docs/ubuntu.md]: Is it Installing? Just a small typo here My friend

**Review by DiegoPino — CHANGES_REQUESTED** [review]
tiny little piggy typo

File: docs/ubuntu.md (hunk 1 above)
Comment (mitchellkeaney) [reply docs/ubuntu.md]: ༼ つ◕(oo)◕༽つ ooops!

changing now!

**Review by DiegoPino — APPROVED** [review]
Approved!

**Comment by DiegoPino** [comment]
@senyzspalding and now you are immortalized into our documentation! Thanks @mitchellkeaney for updating this.

**Pull request merged**